pandas/io/formats/style.py

```diff
@@ -935,19 +969,21 @@ def _highlight_null(v, null_color):
             "background-color: {color}".format(color=null_color) if pd.isna(v) else ""
         )
 
-    def highlight_null(self, null_color="red"):
+    def highlight_null(self, null_color="red", subset=None):
```

Comment:
How is this related to `na_rep`?

Reply:
Actually not related. :joy: 
Just making it consistent with `highlight_min` and `highlight_max`, also for completeness of missing values styling and formatting.

Reply:
Nice idea but would prefer if you take this out then. We try to keep PRs to changing one thing at a time as it helps speed up review process

Reply:
Can always do as a follow up PR

Reply:
Thanks, that is a great advice!